modules/_labs/tabs/react/lib/cursor/hooks.tsx

```diff
@@ -55,15 +55,15 @@ export const useRovingFocus = ({state, events}: CursorModel, elemProps = {}) =>
   return mergeProps(
     {
       onKeyDown(event: React.KeyboardEvent) {
-        Object.keys(orientationKeyMap[state.orientation]).forEach(
-          (key: keyof typeof orientationKeyMap[typeof state.orientation]) => {
-            if (isRTL ? event.key === rightToLeftMap[key] : event.key === key) {
-              const event = orientationKeyMap[state.orientation][key];
-              focusRef.current = true;
-              events[event]?.();
-            }
+        (Object.keys(
+          orientationKeyMap[state.orientation]
+        ) as (keyof typeof orientationKeyMap[typeof state.orientation])[]).forEach(key => {
+          if (isRTL ? event.key === rightToLeftMap[key] : event.key === key) {
+            const event = orientationKeyMap[state.orientation][key];
+            focusRef.current = true;
+            events[event]?.();
           }
-        );
+        });
```

Comment:
`Object.keys` returns `string[]` vs `(keyof T)[]` which is why we have to cast. Sometimes I make a simple utility function with the correct signature. Why couldn't they just update `Object.keys`?